src/server/helpers/achievement.js

```diff
@@ -140,7 +191,13 @@ function testTiers(signal, editorId, tiers) {
 		);
 }
 
-// returns the number of typeRevisions an editor has
+/**
+ * Returns number of revisions of a certain type there are for the specified
+ * editor
+ * @param {string} type - Camelcase name of the type to query
+ * @param {int} editor - Editor id being queried
+ * @returns {int} - Number of revisions of type (type)
+ */
```

Comment:
Unrelated to this PR, but we should probably move this function to a property of the Editor module in -data-js, so we can just do Editor.getTypeRevisions(...) (possibly useful in the WS too).